### ISSUE TITLE
Crashes by empty DecodedData at didReceiveDecodedData:decodedData:fromDevice

### ISSUE THREAD

**Issue by yoi-hibino** [callstack/react-native-socket-mobile#17]
## Environment
Model-D750

## Description
`-(void)didReceiveDecodedData:(SKTCaptureDecodedData*) decodedData fromDevice:(SKTCaptureHelperDevice*) device withResult:(SKTResult) result;`

Scanner sometimes (once a 10-20 continuous scans) returns empty DecodedData with SKTSUCCESS(result). Then it crashes when it constructs NSDictionary with nil value at @{@"data":  [NSString... .
I currently modify code to avoid this issue as follows, but it will be nicer to return some sort of error message with in sendEventWithName. 

```
-(void)didReceiveDecodedData:(SKTCaptureDecodedData*) decodedData fromDevice:(SKTCaptureHelperDevice*) device withResult:(SKTResult) result{
    if (SKTSUCCESS(result) && socketMobileHasListeners) {
        NSString* dataString = [NSString stringWithUTF8String:(const char *)[decodedData.DecodedData bytes]];
        if (dataString == nil) {
            dataString = @"";
        }
        [self sendEventWithName:DecodedData body:@{@"data": dataString}];
    }
}
```

**Comment by Sam-Hoult** [comment]
Solved using this: https://github.com/callstack/react-native-socket-mobile/pull/16

**Comment by ferrannp** [comment]
Merged #16 and released `0.1.1`. I do not have the hardware anymore so please open an issue if something is broken again :)
